robot_skills/src/robot_skills/base.py

```diff
@@ -192,26 +198,35 @@ def move(self, position_constraint_string, frame):
 
         return plan
 
-    def force_drive(self, vx, vy, vth, timeout):
+    def force_drive(self, vx, vy, vth, timeout, ax=float('inf'), ay=float('inf'), ath=float('inf')):
```

Comment:
Do we want to get rid of the distance=speed*timeout?

Reply:
By default no, but  if you want to make things hard on yourself by using the extra arguments I see it as extra freedom so I would say yes